modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -872,7 +872,7 @@ public abstract class Base${schemaName}ResourceImpl
 	}
 
 	protected Map<String, String> addAction(String actionName, Long id, String methodName, Long ownerId, String permissionName, Long siteId) {
-		return ActionUtil.addAction(actionName, getClass(), id, methodName, contextScopeChecker, ownerId, permissionName, siteId, contextUriInfo);
+		return ActionUtil.addAction(actionName, getClass(), contextHttpServletRequest, id, methodName, contextScopeChecker, ownerId, permissionName, siteId, contextUriInfo);
```

Comment:
I think this change is not backwards compatible... If you run rest builder in 7.3 this won't compile because the new method is not there.